templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml

```
@@ -0,0 +1,79 @@
+name: "Frogbot Scan Pull Requests"
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  scan-pull-requests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install prerequisites
+
+      # Uncomment if you have a Pip/Pipenv/Poetry project
+      # - uses: actions/setup-python@v4
+      #   with:
+      #     python-version: "3.x"
+
+      # Uncomment if you have Pipenv project
```

(same for the others)
```suggestion
      # Uncomment if you have a Pipenv project
```

---

done